src/Extension/MemberExtension.php

```diff
@@ -39,4 +47,71 @@ public function getDefaultRegisteredMethod()
     {
         return $this->owner->RegisteredMFAMethods()->byId($this->owner->DefaultRegisteredMethodID);
     }
+
+    public function updateCMSFields(FieldList $fields)
+    {
+        Requirements::javascript('silverstripe/mfa: client/dist/js/bundle-cms.js');
+        Requirements::css('silverstripe/mfa: client/dist/styles/bundle-cms.css');
```

Comment:
This doesn't work in SilverStripe 4.x with PJAX loaded panels. Instead you need to add them to LeftAndMain via extra_requirements_js/css

Reply:
Good catch - I'd been testing by reloading the view directly so hadn't noticed 😞